tachyon/zk/plonk/vanishing/prover_vanishing_argument.h

```diff
@@ -90,6 +92,11 @@ template <typename PCS, typename ExtendedEvals>
                   [](bool result) { return result == false; })) {
     return false;
   }
+  std::vector<Commitment> commitments =
+      prover->pcs().GetBatchCommitments(prover->pcs().batch_commitment_state());
```

Comment:
If you can retrieve `BatchCommitmentState()` from `pcs()`, then this seems redundant.

```suggestion
      prover->pcs().GetBatchCommitments();
```